src/chi/components/steps/steps.scss

```diff
@@ -0,0 +1,78 @@
+@import 'variables';
+@import 'mixins';
+
+.chi {
+
+  .a-steps {
+    padding: 0;
+
+    & > li {
+      color: $brand-color;
+      display: inline-block;
+      font-size: $text;
+      position: relative;
+      text-align: center;
+      width: 20%;
+      z-index: 0;
+
+      &:before {
+        background-color: $text-white;
+        border-radius: 50%;
+        content: '\25CF';
+        display: block;
+        font-size: 1.7rem;
+        height: 1.125rem;
+        line-height: 1rem;
+        margin: 0 auto 0.25rem;
+        width: 1.125rem;
+      }
+
+      &:after {
+        background-color: $brand-color;
+        content: '';
+        height: 0.125rem;
+        left: 50%;
+        position: absolute;
+        top: 0.5rem;
+        width: 100%;
+        z-index: -1;
+      }
+
+      &:last-child {
+        &:after {
+          display: none;
+        }
+      }
+
+      &.-active {
```

Comment:
The white in the active circle is rendering slightly larger than the mock

Current:
![image](https://user-images.githubusercontent.com/130654/48756279-5fe0b900-ec4d-11e8-9dc6-a98b5b0aa4ea.png)
Expected:
![image](https://user-images.githubusercontent.com/130654/48756273-58211480-ec4d-11e8-916b-d842a3229515.png)